Dockerfile

```diff
@@ -6,7 +6,7 @@ COPY . /go/src/github.com/siddontang/go-mysql-elasticsearch
 
 RUN cd /go/src/github.com/siddontang/go-mysql-elasticsearch/ && \
     go build -o bin/go-mysql-elasticsearch ./cmd/go-mysql-elasticsearch && \
-    cp -f ./bin/go-mysql-elasticsearch /go/bin/go-mysql-elasticsearch
-
-ENTRYPOINT ["go-mysql-elasticsearch"]
+    cp -f ./bin/go-mysql-elasticsearch /go/bin/go-mysql-elasticsearch && \
+    apk add --no-cache tini mariadb-client
```

Comment:
I prefer using a separated command for the library.

Reply:
You mean put ʻapk addʻ in an separate RUN command?